script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -91,7 +91,7 @@ I feel a little better after hearing something that I can agree with so wholehea
 [sha:f67e98223db05bf406df4f796542e74c7621aed2]{
 -- Page 18, Offset 4897.
 -- 朝は７時に起きても遅いと文句は言われるし、門限は８時だし。おまけに無断外泊すると厳しく詰問されるなんて、監獄生活みたいだろ？」
-Even if I wake up at seven o'clock in the morning they grouse about me being late, and curfew is at eight o'clock in the evening.
+Even if I wake up at seven o'clock in the morning, they grouse about me being late, and the curfew is at eight o'clock in the evening.
```

Comment:
Don't like first comma, also not sure you need `the` before curfew. More casual.